pkg/ecdsa/tss/protocol_ready.go

```diff
@@ -86,6 +108,30 @@ func readyProtocol(
 
 	switch ctx.Err() {
 	case context.DeadlineExceeded:
+		for _, memberID := range group.groupMemberIDs {
+			memberAddress, err := memberIDToAddress(
+				memberID,
+				group.groupID,
+				pubKeyToAddressFn,
+			)
+			if err != nil {
+				logger.Errorf(
+					"could not convert member ID to address: [%v]",
+					err,
+				)
+				break
+			}
+			_, announcedReadiness := readyMembers[memberAddress]
+			if !announcedReadiness {
```

Comment:
I think it can be simplified a bit:

```
if ok := readyMembers[memberAddress]; !ok {
	logger.Errorf(
		"member [%v] has not announced its readiness for keep [%v]; "+
			"check if keep client for that operator is active and "+
			"connected",
		memberAddress,
		group.groupID,
	)
}
```

Reply:
Done in https://github.com/keep-network/keep-ecdsa/pull/654/commits/e410b4d7c974b8d302c1e77cfea484636da18b37